pkg/common/utils.go

```diff
@@ -251,15 +255,39 @@ func ValidateSnapshotType(snapshotType string) error {
 	}
 }
 
-// ConvertGiBStringToInt64 converts a GiB string to int64
-func ConvertGiBStringToInt64(str string) (int64, error) {
+// ConvertStringToInt64 converts a string to int64
+func ConvertStringToInt64(str string) (int64, error) {
```

Comment:
It may be better to use kubernetes resource quantity parsing instead of rolling your own: https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#Quantity

Reply:
But they do not have our use case. They do not convert these values?

Reply:
I'm not sure I understand. It looks like this function is converting from a string, possibly abbreviated in SI notation, to the raw number right?

Reply:
Yes and we want to support:
```
"1000" -> int64(1000)
"1K"-> int64(1000)
"1M" -> int64(1,000,000)
Maybe "1G" -> int64(1,000,000,000)
```

Reply:
The "K" is not supported in [resource#Quantity](https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#Quantity). But other than that mostly covered. We do not need to support the "K"?